### DISCUSSION TITLE
Thank you / vue conf / Next pinia orm ToDos

### DISCUSSION THREAD

**Discussion by CodeDredd** [CodeDredd/pinia-orm#371] [Announcements]
Hey together!

I wanted to say thank you to all of you who are using this library, are posting issues, are contributing and are starting discussions.
It was a long road till v1 but i am happy that i was able to get all necessary ToDos done.

A great thank you goes to the people who sponsored me! This small things helped me to keep motivated to finish it 😉 ❤️ 
- @tintin10q 
- @svenhue
- @paolodina

Next things i am going to think about is how to include and work together with:
- axiois
- graphql
- vue-query

If someone also wanna talk with me in person then you got your chance at [vue conf Berlin](https://conf.vuejs.de/)

**Comment by eduardodgarciac** [comment]
Hi @CodeDredd, 

I am interested in using this library together with vue-query, by any chance, do you have any examples on how to integrate them? or maybe some tips

**Reply by CodeDredd** [reply]
Hey @eduardodgarciac ,
sry for the late answer. Was busy and ill the last weeks. I am working on best practice examples right now (#999). As soon as i have more time i will release what i think is the beset option right now with pinia-orm until i can write plugins
